app/src/pages/inside/common/itemPathTooltip/itemPathTooltip.jsx

```diff
@@ -16,20 +16,9 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import { TextTooltip } from 'components/main/tooltips/textTooltip';
+import { HistoryLineItemTooltip } from 'pages/inside/logsPage/historyLine/historyLineItem/historyLineItemTooltip';
 
-const getTooltipContent = (testItem = {}) => {
-  const { itemPaths = [], launchPathName = {} } = testItem.pathNames;
-
-  return itemPaths.reduce(
-    (path, item, index) => `${path} \n<b>Parent ${index + 1}:</b> ${item.name}`,
-    `<b>Launch:</b> ${launchPathName.name} #${launchPathName.number}`,
-  );
-};
-
-export const ItemPathTooltip = ({ testItem }) => (
-  <TextTooltip tooltipContent={getTooltipContent(testItem)} />
-);
+export const ItemPathTooltip = ({ testItem }) => <HistoryLineItemTooltip {...testItem} />;
```

Comment:
As this component uses not just for `historyItem` but also for `editToInvestigateModal` this changes breaks tooltip logic in the last one. Please check with @evjlobanova whether we need a new tooltip for `editToInvestigateModal` or not.

Reply:
EditToInvestigateModal is no longer used and will be completely removed after the full modal "Make a decision" implementation, so I think this tooltip can be changed without any side effects.

Reply:
@KatsiarynaTatarynovich 
`EditToInvestigateModal` is no longer used and will be completely removed after the full modal "Make a decision" implementation, so I think this tooltip can be changed without any side effects.

Reply:
I meant `editToInvestigateDefectModal`, sorry for missunderstanding :) Is `editToInvestigateDefectModal` also no longer will be used?

Reply:
Yes, I understand you correctly :) Our defect modals have been completely replaced with the new "Make decision" model.